tartufo/scanner.py

```diff
@@ -336,6 +339,36 @@ def should_scan(self, file_path: str):
             return False
         return True
 
+    @property
+    def config_data(self):
+        return self._config_data
+
+    @config_data.setter
+    def config_data(self, data) -> MutableMapping[str, Any]:
+        self._config_data = data
+        return self._config_data
```

Comment:
I can't quite figure out why this property is needed. Can you explain a bit?

Reply:
I need this as a place holder for the config data, so that I can use it to compute the signature as part of excluded_findings which is accessed without passing inputs.